datafusion/sql/tests/cases/plan_to_sql.rs

```diff
@@ -323,13 +323,6 @@ fn roundtrip_statement_with_dialect() -> Result<()> {
             parser_dialect: Box::new(MySqlDialect {}),
             unparser_dialect: Box::new(UnparserMySqlDialect {}),
         },
-        TestStatementWithDialect {
-            sql: "select * from (select * from j1 limit 10);",
-            expected:
-                "SELECT * FROM (SELECT * FROM `j1` LIMIT 10) AS `derived_limit`",
```

Comment:
If we expand * to all the columns, should we convert back to * for unparser?

I'm not sure should we support this, I delete related test and consider not supported